src/components/Article.jsx

```diff
@@ -1 +1,33 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <>
```

Comment:
why do you need fragment here?

Reply:
I actually don't need it.
Started the return with a fragment and then I forgot to delete it when I have pasted the box div inside it.

I will quickly erase it.

Thank you @naziamoff!